<!--
    This file launches the driver node.
    The "commands" argument is a list and can contain the following in any order and magnitude:
    LINE,SPEED LINE, SPIN, SPEED SPIN, LINE AND SPIN, TRIANGLE, SQUARE, PLUS
-->


<launch>

<arg name="commands"/> 
<arg name="linearSpeed"       default="0.1"/>
<arg name="angularSpeed"      default="0.3"/>
<arg name="pub_topic"         default="/test_driver/cmd_vel"/>

 <!-- Driver Node-->
  <node pkg="cobot_teleop"  name="test_driver" type="test_driver" output="screen">
   <rosparam param="commands" subst_value="True">$(arg commands)</rosparam>
   <rosparam param="linearSpeed" subst_value="True">$(arg linearSpeed)</rosparam>
   <rosparam param="angularSpeed" subst_value="True">$(arg angularSpeed)</rosparam>
   <remap from="/test_driver/cmd_vel" to="$(arg pub_topic)"/>
 </node>

</launch>

